tests/end2end-tests/data_process/movielens_test.sh

```diff
@@ -27,6 +28,13 @@ python3 -m graphstorm.gconstruct.construct_graph --conf-file $GS_HOME/tests/end2
 
 error_and_exit $?
 
+echo "********* Test Homogeneous Graph Optimization ********"
```

Comment:
Let's create a new .sh for homogeneous test, we can call it homogeneous_test.sh.

Reply:
It is wired to put the test in movelens_test.sh.